keyboards/handwired/horrortroll/k552/readme.md

```diff
@@ -0,0 +1,14 @@
+# Redragon K552 Kumara
+
+![K552](https://i.imgur.com/XpNmJpBh.png)
+
+A handwired K552 Kumara. Built using STM32F103RCT6 Mini
+
+* Keyboard Maintainer: [HorrorTroll](https://github.com/HorrorTroll)
+* Hardware Supported: Redragon K552 Kumara and STM32F103RCT6 Mini
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```

Reply:
Done